src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -19241,7 +19241,7 @@ void func_80859C60(PlayState* play, Player* this, void* arg2) {
     PlayerAnimation_Update(play, &this->skelAnime);
 }
 
-void func_80859CA0(PlayState* play, Player* this, s32 arg2) {
+void func_80859CA0(PlayState* play, Player* this, UNK_TYPE arg2) {
```

Comment:
The forward declaration already uses an `UNK_TYPE` so I just made it consistent with that.
It will conflict with that PR either way so I don't mind much about this